docs/modules/data-structures/pages/vector-collections.adoc

```diff
@@ -0,0 +1,494 @@
+= Vector Collection
+:page-enterprise: true
+:page-beta: true
+
+== Intro
+
+The primary object for interacting with the vector storage is a Vector Collection. This collection holds information about the vectors and associated metadata (user value).
+
+A collection consists of one or more indexes, all sharing a common metadata storage. Each index represents a distinct vector space. Vectors from different indexes are stored independently, and indexes can have different configurations.
+
+Conceptually, a vector collection resembles a key-value store. Here, the key is a user-defined unique identifier, and the value is an object containing metadata (user value) alongside several vectors — one for each index.
+
+Users are permitted to store any necessary value for work as metadata (user value). This can be additional characteristics of the source data or even the source data itself.
+
+== Configuration
+
+Collection configuration can be set dynamically during vector collection creation or statically during cluster configuration. Unlike other data structures, the configuration must be set up before the collection can be used. Below is a table listing all available options:
+
+.Collection configuration options
+[cols="1,2,1,1",options="header"]
+|===
+|Option|Description|Required|Default
+
+|name
+|The name of the vector collection.
+|`Required`
+|null
+
+|indexes
+|Information about indexes configuration
+|`Required`
+|null
+|===
+
+.Index configuration options
+[cols="1,2,1,1",options="header"]
+|===
+|Option|Description|Required|Default
+
+|name
+|The name of the vector index.
+|`Required` - for single-index vector collections. `Optional` - for multi-index collection
+|null
+
+|dimension
+|Vectors dimension
+|`Required`
+|N/A
+
+|metric
+|Distance metric used to calculate the distance between two vectors.
+See table <<available-metrics, Available distance metrics.>>
+|`Required`
+|N/A
+
+|max-degree
+|The maximum number of neighbors per node is calculated as max-degree * 2
+|`Required`
+|16
+
+|ef-construction
+|The size of the search queue to use when finding nearest neighbors.
+|`Required`
+|100
+
+|use-deduplication
+|The option for vector deduplication.
+When disabled, each added vector will be treated as a distinct vector in the index, even if it is identical to an existing one. When deduplication is enabled, the index consumes less space, but the time required to add a vector increases.
+|`Required`
+|true
+
+|===
+[#available-metrics]
+.Available distance metrics
+[cols="2,2,2",options="header"]
+|===
+|Name|Description| Score definition
+
+|EUCLIDEAN
+|euclidean distance
+|1 / (1 + squareDistance(v1, v2))
```

Comment:
asciidoc supports math (https://docs.asciidoctor.org/asciidoc/latest/stem/). I wonder if we have it configured, if not this should be code.

Reply:
653f346d2711640a47b0755a7aa6fbf62f4b5d09

Reply:
it does not render correctly in preview

Reply:
[this](https://gitlab.com/antora/antora/-/issues/580) may be the reason

Reply:
Looks like this is a limitation of GitHub preview.

Reply:
I did not mean github preview, I meant https://deploy-preview-1124--hardcore-allen-f5257d.netlify.app/hazelcast/5.5-snapshot/data-structures/vector-collections

Reply:
I cannot see `:stem:` used elsewhere in the doc files; I'd suggest just using code ticks here to be on the safe side.

Reply:
like this?
10d17719c55c9942a5854e4db64a0bf93ccfcdb7